TESTS/mbed-crypto/sanity/main.cpp

```diff
@@ -68,6 +68,25 @@ void test_crypto_asymmetric_encrypt_decrypt(void)
     mbedtls_psa_crypto_free();
 }
 
+void test_crypto_hash_verify(void)
+{
+    psa_algorithm_t alg = PSA_ALG_SHA_256;
+    psa_hash_operation_t operation;
+    unsigned char hash[] = {
```

Comment:
What is this a hash of? Include a comment in the code.

Reply:
OK will do.